### DISCUSSION TITLE
Open field behavior

### DISCUSSION THREAD

**Discussion by saskiad** [AllenNeuralDynamics/aind-data-schema#613] [General]
We currently have modalities of "Behavior videos" and "Trained Behavior." We are preparing for open field, untrained behavior (eg. motor observatory). 
Should we have a modality of "Open field behavior" in which case should we replace "trained behavior" with "headfixed behavior"? 
Should we have a modality of "exploratory behavior" or "free behavior" or "untrained behavior" and then add a field somewhere to indicate "headfixed" vs "open field"?
Other ideas? @dyf @bruno-f-cruz

**Comment by cindypoo** [comment]
I guess trained behavior can be either head-fixed or freely-moving. Open field is a sub-set of freely-moving behavior, that usually is untrained but not always. Maybe "head-fixed" vs "freely-moving" should be at the top level? And other things such as "open field", "maze", "lever-press box", "VR" can be fields? 
Also, unclear if 'trained vs untrained' is easy to interpret, and whats the distinction is with 'conditioned vs unconditioned'. for example Adriend's behavior will be mouse in a box with nothing to do, conditioned with odor, but not trained to do anything.

**Comment by dyf** [comment]
My suggestion is to just have a `behavior` modality and to describe the type of behavior in the `Session`.

**Comment by saskiad** [comment]
Are you saying "Behavior" and then somewhere else we have a different field to specify videos, headfixed, open-field?
This means all physiology is now labeled as behavior because all physiology has behavior videos. Not a problem, per se, but I feel like we'll lose some of the purpose of the modality label...

**Reply by dyf** [reply]
Sorry, I wasn't clear. I'm proposing that there be two modalities:
1. `behavior` (formerly `trained-behavior`, but now would cover headfixed, open field, etc)
2. `behavior-videos`

I worry that the list of behavior subtypes will change quickly and having to rev the schema for every new one will be hard to maintain.
